src/main/java/se/diabol/jenkins/pipeline/DeliveryPipelineView.java

```diff
@@ -111,7 +111,7 @@
     private boolean showAggregatedChanges = false;
     private String aggregatedChangesGroupingPattern = null;
     private String theme = DEFAULT_THEME;
-
+    private int maxNumberOfJobs = -1;
```

Comment:
Maybe we should give it a more descriptive name, such as "maxNumberOfVisiblePipelines" (and update the name of the test cases accordingly)